pom.xml

```diff
@@ -63,6 +67,11 @@
             <artifactId>modelmapper</artifactId>
             <version>${modelMapper.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Can we place it in into plugins? 
```
<build>
    <pluginManagement>
        <plugins>
            <plugin>
....
```

Reply:
When it places in plugin section it doesn't work but I deleted it at all and it still works. -_-